custom/setup.py

```diff
@@ -0,0 +1,53 @@
+import os
+import itertools
+
+from typing import Dict, List
+from setuptools import setup, find_packages
+
+ROOT_PATH = os.path.dirname(__file__)
+PKG_NAME = "mlservermllib"
```

Comment:
I was thinking that `mlserver-mllib` could be nicer but I realise that Python doesn't like hyphens much. Although that seems to be convention for "extension packages". For example, if you search for `pytest` plugins, you can see that they are usually named as `pytest-*`: https://pypi.org/search/?q=pytest-

In the end, it probably boils down to preference. Which one would you go for @FogDong ?